### DISCUSSION TITLE
Add the "don't share the same primary type" rule.

### DISCUSSION THREAD

**Discussion by Carenia** [jynnie/soullocke#14] [Ideas]
I would use this but it doesn't have the option for this rule:

"Often, players will restrict the use of shared types between the teams, meaning that no Pokémon on either team may share the same primary type."

**Comment by jynnie** [comment]
Ohh I see, like you would like to turn on an option that would make it so you couldn't swap in Pokemon that would then lead to shared types?

**Reply by Carenia** [reply]
Yes, please!

We are using this rule for our soullock runs and it would be nice if the website told us if there were any shared primary types. Or, perhaps the website could offer a "suggested" party using all the pokemon we have, that way we can use as many pairs as we have available. If that makes sense.

We have had quite an unlucky run and have a ton of duplicate primary types. So many normals and water primaries. We have broke our brains going through our extra _fifteen_ pairs to try and make our party more than just 3...

**Reply by jynnie** [reply]
Ohhh that's so unlucky. I'll look into it! Currently the tech doesn't pull types, so I'll look into how we could do that for this.

**Reply by FRQZEN98** [reply]
Good morning. Just curious if you are still working on this or have any intention of doing so.
I would love to have the feature to enable the rule of the teams not being able to share any primary types, and perhaps a page that calculates the teams that you could make (selectable pairs that you want to have on the team, selectable that you don't want, and it calculates what other pairs could fit on that team).
Thanks.

**Reply by jynnie** [reply]
Hello! I am planning, at minimum to add a toggle to display types at relevant stages. The calculation page sounds really cool and will it add it to my list as well. To be fully transparent, adding passwords and a mobile friendly version of the site are currently higher on the list for me. And it's been tough with my other work to find time to work on Soullocke. But I will comment/update whenever I start working on it.
